packages/plugin-ext/src/typings/monaco/index.d.ts

```diff
@@ -333,4 +333,5 @@ declare module monaco.modes {
     }
 
     export const SuggestRegistry: LanguageFeatureRegistry<ISuggestSupport>;
+    export const SignatureHelpProviderRegistry: LanguageFeatureRegistry<monaco.languages.SignatureHelpProvider>;
```

Comment:
Is there a reason why you don't use public APIs? We would like to use as less as possible internal APIs, the last upgrade was quite painful.